tests/ext/integrations/curl/distributed_tracing_curl_propagate_asm_event.phpt

```diff
@@ -0,0 +1,40 @@
+--TEST--
+Distributed tracing header tags propagate asm events with curl_exec()
+--SKIPIF--
+<?php if (!extension_loaded('curl')) die('skip: curl extension required'); ?>
+<?php if (!getenv('HTTPBIN_HOSTNAME')) die('skip: HTTPBIN_HOSTNAME env var required'); ?>
```

Comment:
There's no reason to make the this test depend on external services being launched before... Just launch the webserver in the beginning of the test and kill it at the end, like in a destructor.

Reply:
I chose to follow the same approach other tests on the tracer

Reply:
ah sorry I thought this was on appsec